MapboxNavigation/NavigationMapView.swift

```diff
@@ -469,62 +468,101 @@ open class NavigationMapView: MGLMapView, UIGestureRecognizerDelegate {
      */
     public func show(_ routes: [Route], legIndex: Int = 0) {
```

Comment:
@1ec5, do you remember how `legIndex` can be used here? As I can see after adding vanishing route line functionality it's no longer used when creating shape for the route, and the only place I can see where it was used before was to control `public let MBCurrentLegAttribute = "isCurrentLeg"` property in`MGLPolylineFeature`. 

After running few tests I don't see that setting `isCurrentLeg` attribute to `true` or `false` affects anything though.

Reply:
This is functionality that broke when implementing #1307. Any leg other than the current leg is supposed to be dimmed out: #2678. That no longer matters as much for already-passed legs, but it does matter for legs ahead of the current leg.

Reply:
Thanks for clarifying this. In this case probably good idea would be to tackle it in https://github.com/mapbox/mapbox-navigation-ios/issues/2733, fix for it might resolve https://github.com/mapbox/mapbox-navigation-ios/issues/2678 as well.